### ISSUE TITLE
`cargo-modules -V` should print the version

### ISSUE THREAD

**Issue by HongxuChen** [regexident/cargo-modules#10]
`cargo-modules -V` doesn't print the version.

```
$ cargo-modules -V
cargo-modules
```

Surely the version info can be retrieved from `~/.cargo/.crates.toml` however it's a bit inconvenient.

**Comment by regexident** [comment]
Adding proper version printing should be quite straight-forward, I think.

It appears to basically boil down to adding an option `-v`/`—version` via **clap** and then doing:

```rust
let package_name option_env!(“CARGO_PKG_NAME”).unwrap_or(“cargo-modules”);
let package_version = option_env!("CARGO_PKG_VERSION").unwrap_or("unknown");
println!("{} {}", name, version);
```

Would you like to tackle this yourself? I’d be happy to mentor you/answer questions if you have any.

**Comment by HongxuChen** [comment]
@regexident No need, you can add the corresponding code 😄

**Comment by regexident** [comment]
Fixed with d98ba14a4705088eb53179a81edfedafbe8f7e57 aka 0.3.4
